dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/sampling/DefaultSampler.kt

```diff
@@ -0,0 +1,17 @@
+package com.datadog.android.core.internal.sampling
+
+import java.util.Random
+
+internal class DefaultSampler(internal val sampleRate: Float) : Sampler {
```

Comment:
Nitpicking, I'd name that `RateBasedSampler` instead. 
Default doesn't really explain what it does ;)